### DISCUSSION TITLE
NFS csi drivers issue on okd 4.13

### DISCUSSION THREAD

**Discussion by sireeshau472** [okd-project/okd#1733] [General]
Hi Team,

Kindly need a quick assistance on the below issue .

Normal   Scheduled               41m                  default-scheduler  Successfully assigned kube-system/csi-nfs-controller-7b5d474bf9-4t7hv to okdtest-gw89l-worker-easte2-4hqx7
  Warning  FailedCreatePodSandBox  80s (x177 over 41m)  kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = creating pod sandbox with name "k8s_csi-nfs-controller-7b5d474bf9-4t7hv_kube-system_132b26ef-195f-4d0f-8dca-4734d0676c56_0": initializing source docker://registry.ci.openshift.org/origin/4.13-2023-07-24-172407@sha256:3de9b15fccdd80fd269ddf3e6b5fdf53c9596db9a2c38b78ec1b98a98d73513c: reading manifest sha256:3de9b15fccdd80fd269ddf3e6b5fdf53c9596db9a2c38b78ec1b98a98d73513c in registry.ci.openshift.org/origin/4.13-2023-07-24-172407: received unexpected HTTP status: 500

**Comment by vrutkovs** [comment]
>4.13-2023-07-24-172407


This version was never released as a stable, so its using CI registry. See README.md:
>Please note that each nightly release is pruned after 72 hours. If the nightly that you installed was pruned, the cluster may be unable to pull necessary images and may show errors for various functionality (including updates)
